recipes/example/meta.yaml

```diff
@@ -93,3 +93,4 @@ extra:
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
     - LandoCalrissian
+    
```

Comment:
Can you revert the change in the example? We want to reduce the noise on things that should not change

Reply:
Sure!